roles/pulibrary.pulmap/meta/main.yml

```diff
@@ -1,3 +0,0 @@
----
-dependencies:
```

Comment:
This dependency is what ensures that environment variables are set for the `deploy` user.  I would please recommend that this Role be kept, or that `rails-app` be used instead.

Reply:
`rails-app` role added.